setup.cfg

```diff
@@ -52,7 +52,7 @@ doc =
     packaging
     pydot >= 1.2.3
     pydotplus
-    sphinx >= 1.5.3, < 3
+    sphinx >= 1.5.3
```

Comment:
when the new API of sphinx started? as early as 1.5.3?

Reply:
1.8

Reply:
Then
```suggestion
    sphinx >= 1.8
```
I guess?

Reply:
Yup. I've got a few commits I haven't pushed yet. I was just using this to test for the moment. I would say let's make this a real PR when we depend on 1.5.0.